src/main/java/br/ufes/inf/nemo/frameweb/vp/FrameWebPlugin.java

```diff
@@ -69,6 +69,9 @@ public class FrameWebPlugin implements VPPlugin {
   /** Indicates if the Plug-in Settings Dialog is open. */
   private boolean pluginSettingsDialogOpen = false;
 
+  /** Indicates if the Select DAO Dialog is open. */
+  private boolean SelectDAODialogOpen = false;
```

Comment:
```suggestion
  private boolean selectDAODialogOpen = false;
```

(Coding standards, attributes begin with lower case)